### ISSUE TITLE
Localize aria-label

### ISSUE THREAD

**Issue by rugk** [TinyWebEx/MessageHandler#3]
It should be recommended to translate the aria-labels (via https://github.com/TinyWebEx/Localizer). Should be just some HTML markup…
